README.md

```diff
@@ -1,70 +1,48 @@
-# Getting Started with Create React App
+## COVID 19 top 20 African cases
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> This is a website that will display top 20 African countries that are most affected by COVID. A rest API has been used to fetch the COVID data of all the listed countries.
 
-## Available Scripts
+## Screenshot
 
-In the project directory, you can run:
+![screenshot](/src/components/img/covid-app.png))
 
-### `npm start`
+## Live Demo
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- [COVID African Data](https://victor-covid-19-app.netlify.app/)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Loom Video
 
-### `npm test`
+- [Project Loom Video](https://www.loom.com/share/8972d573f2f14cc6be9b05f234a41bb0/../../../../package-lock.json)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Built With
 
-### `npm run build`
+- React
+- Redux
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
```

Comment:
Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [ ] _How to set **test** this project_ **should be detailed, clear, and well** organized.
- [ ] OPTIONAL
Kindly ensure that your ReadMe is _typo-free_. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)